homework-4 PR

najahizquierdo opened the pull request:
Need to figure out why my repository isnt allowing me to push from the terminal.

najahizquierdo committed:
Add response check, regression. plots not working so i removed them

Need to figure out why my repository isnt allowing me to push from the terminal.

najahizquierdo commented:
@dafrenchyman @brian004900

brian004900 commented on homework4/assignment4.py:
```diff
@@ -0,0 +1,87 @@
+        return True
+    else:
+        return False
+
```
This def can determine if the response is con or cat.
The predictors can also be resolved this way.

brian004900 commented on homework4/assignment4.py:
```diff
@@ -0,0 +1,87 @@
+
+    # linear regression
+    if response == "continuous":
+        for a, col in enumerate(X.T):
```
p-value & t-score are for continuous predictors only.

brian004900 commented on homework4/assignment4.py:
```diff
@@ -0,0 +1,87 @@
+            col = X[:, a]
+            feature_name = data.feature_names[a]
+            predictor = statsmodels.api.add_constant(col)
+            logi_regression_model = statsmodels.api.Logit(
```
successfully did logistic regression.

najahizquierdo committed:
final upload

not fully completed. alot of my code broke and the logic was off with most. Hopefully it's somewhat decent :(
